### PR TITLE
Change template_path to template_paths in documentation examples

### LINKED ISSUE #1371: `template_path` still appearing in docs, despite rename
Various places in the docs still say to use `template_path`, but it's been renamed to `template_paths`.

### PR DESCRIPTION
**Pull request by willingc**
Closes #1371

### CODE REVIEW

File: docs/api_examples/template_path/make_html.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 """
 this script builds html files with either classic or classic_clone templates
+
+Note: nbconvert 6.1 changed ``template_path`` to ``template_paths``
```
Comment (MSeal) [inline docs/api_examples/template_path/make_html.py:4]: Technically 6.0 changed it, but this is fine

**Review by MSeal — APPROVED** [review]

**Comment by MSeal** [comment]
Thanks @willingc

File: docs/api_examples/template_path/make_html.py (hunk 1 above)
Comment (willingc) [reply docs/api_examples/template_path/make_html.py:4]: You know that I'm going to go back and change this ;-)

Comment (MSeal) [reply docs/api_examples/template_path/make_html.py:4]: Well I didn't want to get in the way of improvements merging :)

**Pull request merged**